### PR TITLE
buttons and links

### PR DESCRIPTION
**Pull request by AnuragSahu11**
image

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+.btn-close {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  color: inherit;
+}
+
+/* button vertical */
+
+.btn-vertical :nth-child(n) {
```
Comment (saadahmadhere) [inline components/button.css]: I am skeptic if using :nth-child is a good practice. Could you go for more specific selectors. Like, if you want to select all the children of an element, you can use '> *' selector.

File: components/button.css [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  width: 100%;
+}
+.btn-vertical :nth-child(n + 2) {
+  margin-top: 5px;
+}
+
+/* button horizontal */
+
+.btn-horizontal {
```
Comment (saadahmadhere) [inline components/button.css:172]: This is empty as of now.

**Review by saadahmadhere — COMMENTED** [review]
Looks good to me.

File: components/button.css (hunk 1 above)
Comment (AnuragSahu11) [reply components/button.css]: Yes I will make the changes

File: components/button.css (hunk 2 above)
Comment (AnuragSahu11) [reply components/button.css:172]: Yup, I will see to that

File: components/button.css [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+}
+
+/* Buttons */
+
+/* button with size */
+
+.btn-large {
+  font-weight: 700;
+  cursor: pointer;
+  font-size: 2.1rem;
```
Comment (prachi-sahani) [inline components/button.css:38]: use consistent units

File: components/button.css [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+}
+.btn-small {
+  font-weight: 700;
+  cursor: pointer;
+  font-size: 1.4rem;
+  background-color: var(--primary);
+  width: 91px;
+  height: 32px;
+  padding: 8px 16px;
+  min-width: fit-content;
```
Comment (prachi-sahani) [inline components/button.css:69]: redundant? (since you have already defined fixed width and height)

**Review by prachi-sahani — COMMENTED** [review]
apart from the points mentioned, code looks fine.

**Pull request merged**